packages/mask/src/plugins/Avatar/Application/NFTListDialog.tsx

```diff
@@ -206,19 +205,21 @@ export function NFTListDialog(props: NFTListDialogProps) {
                         onChangePage={onChangePage}
                         tokens={tokens}
                         children={
-                            state !== SocketState.done && tokens.length === 0
-                                ? LoadStatus
-                                : currentPage === Application_NFT_LIST_PAGE.Application_nft_tab_eth_page
-                                ? error && tokens.length
-                                    ? Retry
-                                    : undefined
-                                : AddCollectible
+                            tokens.length === 0
+                                ? state !== SocketState.done
+                                    ? LoadStatus
+                                    : currentPage === Application_NFT_LIST_PAGE.Application_nft_tab_eth_page
+                                    ? error && tokens.length
+                                        ? Retry
+                                        : undefined
+                                    : AddCollectible
+                                : undefined
```

Comment:
plz extract the expression as function, and unfold nested ternary expression to if-else structure